scripts/generate_unicode_data.cr

```diff
@@ -204,15 +204,11 @@ body.each_line do |line|
 end
 
 downcase_ranges = case_ranges entries, &.downcase
-downcase_one_ranges, downcase_ranges = downcase_ranges.partition { |r| r.delta == 1 }
+downcase_one_ranges, _ = downcase_ranges.partition { |r| r.delta == 1 }
 
 upcase_ranges = case_ranges entries, &.upcase
 upcase_ranges.select! { |r| r.delta != -1 }
 
-alternate_ranges = alternate_ranges(downcase_one_ranges)
```

Comment:
This is used in the ECR template. That's why the tool doesn't see it.

Reply:
Thanks! I knew I was missing something there...